scripts/sync_data.py

```diff
@@ -59,6 +65,32 @@ def download_contrarianmba_airtable_records():
         f.write("\n")
 
 
+def download_product_image(product_id, force=False):
```

Comment:
Put this function below `download_product_images`, because that one is called first